### PR TITLE
Added references for smt (solvers)

### PR DESCRIPTION
**Pull request by SundermannC**

### CODE REVIEW

File: literature-cleaned.bib [hunk 1]
```diff
@@ -5261,6 +5271,15 @@ @article{AGA:TSE13
 	year = 2013
 }
 
+@inproceedings{CGSS:TACAS13,
+  title= {{The mathsat5 smt solver}},
```
Comment (pmbittner) [inline literature-cleaned.bib:5275]: capitalization

File: literature-cleaned.bib [hunk 2]
```diff
@@ -5261,6 +5271,15 @@ @article{AGA:TSE13
 	year = 2013
 }
 
+@inproceedings{CGSS:TACAS13,
+  title= {{The mathsat5 smt solver}},
+  author={Cimatti, Alessandro and Griggio, Alberto and Schaafsma, Bastiaan Joost and Sebastiani, Roberto},
+  booktitle = TACAS,
+  pages={93--107},
+  year= 2013,
+  organization={Springer}
```
Comment (pmbittner) [inline literature-cleaned.bib:5280]: Change `organization` to `publisher` and remove brackets? `publisher` and `address` are missing. They should be always part of inproceedings.

File: literature-cleaned.bib [hunk 3]
```diff
@@ -8045,6 +8064,19 @@ @article{BFL+11
 	year = 2011
 } 
 
+@article{DB:CACM11,
+  title = {{Satisfiability modulo theories: introduction and applications}},
```
Comment (pmbittner) [inline literature-cleaned.bib:8068]: capitalization

File: literature-cleaned.bib [hunk 4]
```diff
@@ -9110,6 +9142,15 @@ @inproceedings{SBFF:HICSS09
 	year = 2009
 }
 
+@inproceedings{DB:SBFM09,
+  title = {{Satisfiability modulo theories: An appetizer}},
```
Comment (pmbittner) [inline literature-cleaned.bib:9146]: capitalization

File: literature-cleaned.bib [hunk 5]
```diff
@@ -9110,6 +9142,15 @@ @inproceedings{SBFF:HICSS09
 	year = 2009
 }
 
+@inproceedings{DB:SBFM09,
+  title = {{Satisfiability modulo theories: An appetizer}},
+  author = {De Moura, Leonardo and Bj{\o}rner, Nikolaj},
+  booktitle = SBFM,
+  pages = {23--36},
+  year = 2009,
+  organization= Springer
```
Comment (pmbittner) [inline literature-cleaned.bib:9151]: see above

**Review by pmbittner — CHANGES_REQUESTED** [review]
By accident, I made my comments to `literature-cleaned.bib` but the comments apply to `literature.bib` and are valid nevertheless.

**Review by pmbittner — APPROVED** [review]

**Comment by pmbittner** [comment]
Travis is still failing:

**Comment by pmbittner** [comment]
Travis is still failing with:

    > MYshort.bib
    [ERROR] MYshort.bib
    Variable not found: 'aiedam'
    The command ".travis/validate.py" exited with 1.

**Comment by SundermannC** [comment]
> Travis is still failing with:
> 
> ```
> > MYshort.bib
> [ERROR] MYshort.bib
> Variable not found: 'aiedam'
> The command ".travis/validate.py" exited with 1.
> ```
Seems to be caused by the master branch and unrelated to the changes. Fix it on master after merging to reduce conflicts?

**Comment by pmbittner** [comment]
Oops, yes. Seems as if @tthuem caused the failure. :D I agree, merge and fix then.

**Pull request merged**